### ISSUE TITLE
Edit help text to make clear when some codons are excluded by observedMutations

### ISSUE THREAD

**Issue by ssnn-airr** [immcantation/shazam#108]
**[Original report](https://bitbucket.org/kleinstein/shazam/issues/108) by Steven Kleinstein (Bitbucket: [skleinstein](https://bitbucket.org/skleinstein), GitHub: [skleinstein](https://github.com/skleinstein)).**

----------------------------------------

Edit help to make clear the behavior in the description below (from Jason)

...a mutation that introduces a stop codon, which `observedMutations` considers neither a replacement nor silent mutation. For example:
 SEQUNCE_IMGT> AAA
GERMLINE_IMGT> TCA

The stop is introduced when the second A is considered as a mutation, because mutations are counted in the germline context individually. So TAA vs TCA and ACA vs TCA are both considered, with the TAA causing the exclusion of the codon.

Also, ... The codon would be excluded if the germline codon is ambiguous, .... For example:
 SEQUNCE_IMGT> TGG
GERMLINE_IMGT> G.G
So, long story short, there are few different cases where codons are excluded from the mutation counts due to stop codons or being unable to resolve replacement vs silent. Those cases are what's leading to the differences between nucleotide mismatches and mutation counts.
